client/components/LeftDiv.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import ReviewTable from './ReviewTable.jsx';
+
+
+class LeftDiv extends React.Component {
```

Comment:
This might make sense to rename as Header or something more content related and you could do this in your App component as you already get reviews in there so you wouldn't have to make a second api call. but certainly is up to preference.